Systematics/test/workspaceStd.py

```diff
@@ -138,16 +139,34 @@
                            VarParsing.VarParsing.varType.bool,
                            'doFiducial'
                            )
+customize.options.register('doBJetsAndMET',
```

Comment:
These options are no longer needed in workspaceStd. I'm not sure if @threiten already removed these but they are still showing up in github, or if they are really still there